src/components/QuickOpenModal.css

```diff
@@ -11,3 +11,8 @@
 .theme-dark .result-item .subtitle .highlight {
   color: white;
 }
+
+.loadIndicator {
```

Comment:
How about `loading-indicator`? CSS rules generally are not camel case

Reply:
I'll fix this right away